### PR TITLE
kubectxwin: Add version 0.1.1

### PR DESCRIPTION
**Pull request by gracefullight**
https://github.com/lukesampson/scoop/issues/3510

- [kubectx](https://github.com/ahmetb/kubectx)
- [kubectxwin](https://github.com/thomasliddledba/kubectxwin)

### CODE REVIEW

File: bucket/kubectxwin.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+    "homepage": "https://github.com/thomasliddledba/kubectxwin",
+    "description": "Windows Version of the powerful tool kubectx.",
```
Comment (Ash258) [inline bucket/kubectxwin.json]:
```suggestion
    "description": "Kubectl(1) context manager and switcher.",
```

or

```suggestion
    "description": "Kubernetes cluster switcher.",
```

I f

File: bucket/kubectxwin.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    "license": "MIT",
+    "version": "0.1.1",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/thomasliddledba/kubectxwin/releases/download/0.1.1/kubectxwin.exe",
+            "hash": "sha1:34703BCB9B16865541E58524B596B1AEC0E4E3D0"
```
Comment (Ash258) [inline bucket/kubectxwin.json]: Sha1 hashes are provided somewhere? Or you just generated it using sha1?

Comment (gracefullight) [reply bucket/kubectxwin.json]: I just generated 😂

**Comment by gracefullight** [comment]
> I think it could be named `kubectx`. But I am not kubectx user so I cannot properly tell if this repository is official or it needs to be distinguish.

The official Kubectx provides [only bash script version](https://github.com/ahmetb/kubectx/blob/master/kubectx), so I think both libraries are managed separately

**Pull request merged**